web-app/src/pages/Login.vue

```diff
@@ -81,5 +91,6 @@ export default class Login extends Vue {
 
 .container {
   background: linear-gradient($orange-1, $orange-2, $accent);
+  // background: linear-gradient(19deg, #ede7e7 0%, #e7e2f7 100%);
```

Comment:
```suggestion
```

Reply:
Doesn't make any difference. It is only color palette and this was approved by @mblaze

Reply:
it's useless here. Move this color palette to `$background: linear-gradient(19deg, #ede7e7 0%, #e7e2f7 100%);` in `src\css\quasar.variables.scss`

Reply:
Ok